test/gtest/ucp/test_ucp_rma.cc

```diff
@@ -71,8 +71,8 @@ class test_ucp_rma : public test_ucp_memheap {
 
             /* Memory type put/get is fully supported only with new protocols */
             if (!enable_proto() &&
-                (!UCP_MEM_IS_ACCESSIBLE_FROM_CPU(pairs[i][0]) ||
-                 !UCP_MEM_IS_ACCESSIBLE_FROM_CPU(pairs[i][1]))) {
+                (!UCP_MEM_IS_HOST(pairs[i][0]) ||
+                 !UCP_MEM_IS_HOST(pairs[i][1]))) {
```

Comment:
why ? wouldn't it pass?

Reply:
it fails for managed memory as follow because it can't register for managed memory

```

 dcx/test_ucp_rma.get_blocking/0 <dc_x,cuda_copy,rocm_copy/flush_worker>
[     INFO ] host->host 100x94 100x91 100x947 100x1107 100x22073 100x112621 100x26987 100x799262 25x4044769 5x19803637
[1605361783.692263] [swx-dgx01:10611:0]       rma_send.c:342  UCX  ERROR remote memory is unreachable (remote md_map 0x0)
[1605361783.692294] [swx-dgx01:10611:0]      ucp_test.cc:209  UCX  ERROR operation returned error: Destination is unreachable
ucp/test_ucp_memheap.cc:96: Failure
Failed
data validation failed
[     INFO ] host->cuda-managed 100x122
ucp/test_ucp_memheap.cc:96: Failure
Failed
data validation failed
[     INFO ] cuda-managed->host 100x98
[1605361783.693906] [swx-dgx01:10611:0]       rma_send.c:342  UCX  ERROR remote memory is unreachable (remote md_map 0x0)
[1605361783.693916] [swx-dgx01:10611:0]      ucp_test.cc:209  UCX  ERROR operation returned error: Destination is unreachable
ucp/test_ucp_memheap.cc:96: Failure
Failed
data validation failed
[     INFO ] cuda-managed->cuda-managed 100x114
ucp/test_ucp_memheap.cc:96: Failure
Failed
data validation failed
[     INFO ] host->host map_nb 100x126
common/test.cc:332: Failure
Failed
Got 4 errors and 0 warnings during the test
[     INFO ] < rma/rma_send.c:342 remote memory is unreachable (remote md_map 0x0) >
[     INFO ] < ucp/ucp_test.cc:209 operation returned error: Destination is unreachable >
[     INFO ] < rma/rma_send.c:342 remote memory is unreachable (remote md_map 0x0) >
[     INFO ] < ucp/ucp_test.cc:209 operation returned error: Destination is unreachable >
[  FAILED  ] dcx/test_ucp_rma.get_blocking/0, where GetParam() = dc_x,cuda_copy,rocm_copy/flush_worker (1076 ms)
[----------] 1 test from dcx/test_ucp_rma (1076 ms total)

[----------] Global test environment tear-down
[==========] 1 test from 1 test case ran. (1076 ms total)
[  PASSED  ] 0 tests.
[  FAILED  ] 1 test, listed below:
[  FAILED  ] dcx/test_ucp_rma.get_blocking/0, where GetParam() = dc_x,cuda_copy,rocm_copy/flush_worker
```